src/routes/project/TaskRow.svelte

```diff
@@ -85,7 +111,12 @@
 	.task:last-of-type {
 		border-bottom: 0;
 	}
-
+	.task__detail-hidden {
+		opacity: 0;
+		height: 0;
+		overflow: hidden;
+		white-space: nowrap;
+	}
```

Comment:
Look, we, in fact want this node and the `.task__detail-name` to be absoulutelly identical, except this one should be invisible. So, a better way to achieve it would be something like this:
```css
.task__detail-name, .task__detail-hidden {
    /* ... all the styles from .task__detail-name... */
}
.task__detail-hidden {
    viability: hidden;
}
```